microsoft-edge/extensions/publish/marketing-guidelines.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Microsoft Edge Add-ons Marketing Guidelines
+description: Microsoft Edge Add-ons Marketing Guidelines.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/13/2025
+---
+# Microsoft Edge Add-ons Marketing Guidelines
+
+
+<!-- ====================================================================== -->
+## Purpose
+
+These guidelines outline how to use Microsoft Edge brand assets across websites, documentation, tutorials, and other services or integrations.  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
+
+
+<!-- ====================================================================== -->
+## Microsoft Edge Add-ons Badge
+
+Microsoft Edge Add-ons badge is a visual asset designed to help you promote Edge Add-ons and build user trust.
+
+Microsoft Edge Add-ons badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader Microsoft Store, the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons and extensions that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser
+
+Unlike generic Microsoft Store badges — the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your add-ons such as extensions, webapps directly to the Edge ecosystem
+
+
+<!-- ====================================================================== -->
+#### Minimum sizes
+
+* To ensure the badge text remains readable, display it at a minimum size 32px.  Avoid using smaller sizes.  When scaling the badge, always maintain its aspect ratio—do not stretch or compress it.  This helps preserve visual quality and reinforces your product's Credibility
+
+These badges are available in 41 languages.  Please follow the required specifications for image usage, fonts, and color palette.  Detailed usage examples and guidelines are available in the downloadable PDF.
+
+[Add working Download button: Microsoft Store Badge Guidelines (PDF, English)]
```

Comment:
which website to put .zip file & .pdf file for downloading?  
There was some discussion for a previous PR.

@captainbrosset @leahmsft @G-u-r-j-e-e-t

Reply:
Where to put the couple download files for users to download?  Similar to the solution we used w/ Utkarsh, you (or I, if I had the two files) could _create a PR in the extensions repo_, to add a dir like /badges/ or /assets/:

https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension-samples

per its Readme https://github.com/microsoft/MicrosoftEdge-Extensions, that repo contains:
* Code
* Issues
* Discussions

@G-u-r-j-e-e-t

Reply:
closing Outdated comment.
highly likely that as a good solution, we'll upload PDF & ZIP to Extensions repo eg in new /assets/ dir, comparable to /TestCrxPackages/ dir precedent, but w/ better dir structure.

@G-u-r-j-e-e-t @ut-karsh @leahmsft @captainbrosset